documentation/docs/05-load.md

```diff
@@ -264,6 +219,56 @@ Setting the same header multiple times (even in separate `load` functions) is an
 
 You cannot add a `set-cookie` header with `setHeaders` — use the [`cookies`](/docs/types#sveltejs-kit-cookies) API in a server-only `load` function instead.
 
+### Shared-only inputs
```

Comment:
The term "shared-only" was confusing to me, since I don't think we use this term to refer to `+page.js`/`+layout.js` `load`s anywhere else. Is there something more descriptive?

Reply:
I like 'universal load' but 'universal inputs' sounds like it applies everywhere. Given that the only one is `data` I wonder if the better solution is to restructure this section slightly instead